src/fundus/publishers/us/the_nation_parser.py

```diff
@@ -14,12 +22,14 @@
 
 class TheNationParser(ParserProxy):
     class V1(BaseParser):
+        VALID_UNTIL = date(2023, 7, 22)
+
         # There is a known issue preventing lxml from extracting text content within
         # the specified summary node.
         # This is due to invalid XML provided by The Nation.
         # Currently(lxml 4.9.3), lxml does not accept p tags within any heading (h*) tag.
         # The "correct" selector would be ".article-header-content > h2 > p"
-        _summary_selector = CSSSelector(".article-header-content > h2")
+        _summary_selector: Selector = CSSSelector(".article-header-content > h2")
```

Comment:
We could make this typehint more precise and annotate it directly with `CSSSelector`.

Reply:
The problem here is that mypy will complain that we overwrite CSSSelector wit XPath.

Reply:
Yes, my bad.